docs/access/ssh.md

```diff
@@ -174,6 +174,56 @@ ssh -A cscsusername@ela.cscs.ch
 ssh daint.cscs.ch
 ```
 
+## SSH tunnel to a service on Alps compute nodes via ela
+
+If you have a server listening on a compute node in an Alps cluster and want to reach it from your local computer, you can do the following: allocate a node, start your server bound to `localhost`, open an SSH tunnel that jumps through `ela` to the cluster, then use `http://localhost:PORT` locally.
+Details on how to achieve this are below.
+
+Before starting, make sure you:
```

Comment:
I forgot that you need an empty line before the list. Otherwise the list contents will just be lumped into one big paragraph.
```suggestion
Before starting, make sure you:

```